spec/models/sample_spec.rb

```diff
@@ -43,6 +43,12 @@
         expect(build(:sample, species: nil)).not_to be_valid
       end
     end
+
+    describe 'retention_instruction' do
+      it 'has a retention_instruction' do
```

Comment:
What happens if the retention instruction received is invalid?

Reply:
I was under the impression that `/v1/reception` endpoint was used only by `tol-lab-share`. There are [strict validations](https://redpanda.uat.psd.sanger.ac.uk/console/) when this message is consumed, but not when it is pushed to traction. Although it is not pushing invalid values from `tol-lab-share` currently, there's nothing preventing any other developments in `tol-lab-share` (or any other component that pushes HTTP calls to `/v1/reception` endpoint from pushing an invalid retention instruction. 

I will add some validations here.